commit/metrics/prom.go

```diff
@@ -46,6 +49,22 @@ var (
 		},
 		[]string{"chainID", "state", "type"},
 	)
+	promMerkleProcessorRmnReportLatency = promauto.NewHistogramVec(
+		prometheus.HistogramOpts{
+			Name:    "ccip_commit_merkle_processor_rmn_report_latency",
+			Help:    "This metric tracks the client-observed latency of building an full RMN report with signatures",
+			Buckets: prometheus.DefBuckets,
```

Comment:
Most of the other metrics rely on buckets in millis instead of seconds. Mixing seconds with millis might lead to some confusion across devs. Ideally we should follow `prometheus.DefBuckets` everywhere, but considering that we have millis everywhere I would preferably follow a single standard.

https://github.com/smartcontractkit/chainlink/blob/78d548d6101afd7fc82c2568a3665b4b57d5b1eb/core/services/ocr2/plugins/promwrapper/plugin.go#L65
https://github.com/smartcontractkit/chainlink/blob/78d548d6101afd7fc82c2568a3665b4b57d5b1eb/core/services/functions/listener.go#L74
https://github.com/smartcontractkit/chainlink/blob/78d548d6101afd7fc82c2568a3665b4b57d5b1eb/common/txmgr/broadcaster.go#L50